src/smart-components/role/add-role-permissions/add-role-permission-wizard.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState, useEffect } from 'react';
+import AddRolePermissionView from './add-role-permission-view';
+import AddRolePermissionSummaryContent from './add-role-permissions-summary-content';
+import PropTypes from 'prop-types';
+import { WarningModal } from '../../common/warningModal';
+import { useHistory } from 'react-router-dom';
+import { Wizard } from '@patternfly/react-core';
+import { updateRole } from '../../../helpers/role/role-helper';
+
+const AddRolePermissionWizard = ({ role }) => {
+  console.log('Testing out my role as soon as the Wizard loads: ', role);
+  const [selectedPermissions, setSelectedPermissions] = useState([]);
+  const [cancelWarningVisible, setCancelWarningVisible] = useState(false);
+  const [currentRole, setCurrentRole] = useState({});
+  const [currentRoleID, setCurrentRoleID] = useState('');
+  const history = useHistory();
+
+  const setSelectedRolePermissions = (selected) => {
```

Comment:
Do we need this function? It's just a wrapper function for `setSelectedPermissions`

Reply:
@arivepr^